src/v0/destinations/braze/util.js

```diff
@@ -518,8 +518,18 @@ function setExternalId(payload, message) {
   return payload;
 }
 
-function setAliasObjectWithAnonId(payload, message) {
-  if (message.anonymousId) {
+function setAliasObject(payload, message) {
+  const integrationsObj = getIntegrationsObj(message, 'BRAZE');
```

Comment:
suggestion: does this need to be all caps? There's a mix of cases for object name, but lowercase seems more common.

```suggestion
  const integrationsObj = getIntegrationsObj(message, 'braze');
```

Reply:
We use canonical names to resolve mix of casing for integration Object inputs  [ref1](https://github.com/rudderlabs/rudder-transformer/blob/673311f418b387aa17b6a2e0de6b6dce2871c672/src/constants/destinationCanonicalNames.js#L143) , 
[ref2](https://github.com/rudderlabs/rudder-transformer/blob/673311f418b387aa17b6a2e0de6b6dce2871c672/src/v0/util/index.js#L1005)

Reply:
Ah ok! Thanks for the extra context.